_includes/footer.html

```diff
@@ -1,3 +1,25 @@
+<dialog id="splash">
+<p>If you’re using WPE WebKit, or are considering doing so, <strong>please take our brief user survey!</strong> Your input will help us make WPE WebKit better for you.</p>
+<div>
+<button is="dis-misser" id="dismiss-Y">Yes</button>
```

Comment:
These won't work in WebKit based browsers (is="" see https://webkit.org/standards-positions/#position-97).

We'd have to change these to be like `<dis-misser><button id="dismiss-Y"></dis-misser>` or to just querySelector and handle them

Reply:
```javascript
// if you just change to this it will work...
document.querySelectorAll('button[is="dis-misser"]').forEach(el => {
	el.addEventListener("click", () => {
		splash.close();
		storedInfo.status = el.getAttribute('id').replace('dismiss-','');
		console.log(storedInfo);
		localStorage.setItem(flagName, JSON.stringify(storedInfo));
		if (storedInfo.status == "Y") surveyRedirect();
	});
})
```

Reply:
Fixed.  Thanks for the suggested code; worked a treat.